tsl/test/sql/CMakeLists.txt

```diff
@@ -104,7 +101,11 @@ endif(CMAKE_BUILD_TYPE MATCHES Debug)
 if((${PG_VERSION_MAJOR} GREATER_EQUAL "14"))
   if(CMAKE_BUILD_TYPE MATCHES Debug)
     list(APPEND TEST_FILES chunk_utils_internal.sql)
+    list(APPEND TEST_TEMPLATES continuous_aggs.sql.in
+         continuous_aggs_deprecated.sql)
   endif()
+  list(APPEND TEST_FILES compression.sql compression_upd_del.sql
```

Comment:
nit: suggest to rename the new test file to `compression_update_delete`

Reply:
Done. Thanks.